hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseOptions.java

```diff
@@ -137,4 +138,28 @@ public static synchronized HbaseOptions instance() {
                     null,
                     ""
             );
+
+    public static final ConfigOption<Integer> HBASE_VERTEX_TABLE_REGION_NUM =
+            new ConfigOption<>(
+                    "hbase.hbase_vertex_table_region_num",
+                    "The number of partitions of the HBase vertex table, it can be set to the number of pre-split regions.",
+                    nonNegativeInt(),
+                    1
+            );
+
+    public static final ConfigOption<Integer> HBASE_EDGE_TABLE_REGION_NUM =
+            new ConfigOption<>(
+                    "hbase.hbase_edge_table_region_num",
+                    "The number of partitions of the HBase edge table, it can be set to the number of pre-split regions.",
+                    nonNegativeInt(),
+                    1
+            );
+
+    public static final ConfigOption<Boolean> HBASE_ENABLE_PARTITION =
+            new ConfigOption<>(
+                    "hbase.hbase_enable_partition",
+                    "Is pre-split regions enabled for HBase.",
+                    disallowEmpty(),
+                    false
```

Comment:
If we use hash prefix for logic rowkey design, we should **enable it** by default